policy/SuprimecamMapper.paf

```diff
@@ -572,17 +527,13 @@ datasets: {
         tables:        raw
         tables:        raw_visit
     }
+    # dataset defined in obs_base modified in this mapper as follows:
     forced_src_schema: {
         template:      "schema/forced_src.fits"
-        python:        "lsst.afw.table.SourceCatalog"
-        persistable:        "ignored"
-        storage:    "FitsCatalogStorage"
     }
+    # dataset defined in obs_base modified in this mapper as follows:
```

Comment:
This `python` entry is incorrect.  It should point to a class in `meas.base`, not `pipe.tasks`.  Hopefully that change will bring it into agreement with `obs_base`.